README.md

```diff
@@ -24,7 +24,7 @@ As a Mi A1 user, I was stuck with Stock Oreo 8.0.0 without Oreo custom ROMs when
 
 ## ⭐ Features
 
--   **Currently not working!** ~~Different Pixel Launcher choices (Rootless Launcher, Lawnchair, Ruthless Launcher, Customized Pixel Launcher, stock Pixel Launcher)~~
+-   **Currently being re-worked** ~~Different Pixel Launcher choices (Rootless Launcher, Lawnchair, Ruthless Launcher, Customized Pixel Launcher, stock Pixel Launcher)~~
```

Comment:
I think this shows we are working on one of the biggest features of the module

Reply:
True.